src/components/HoverAnimated.test.tsx

```diff
@@ -71,7 +71,7 @@ test('animates hovering', async () => {
 
     const element = getByTestId('element');
     
-    fireEvent.mouseOver(element);
+    fireEvent.mouseEnter(element);
```

Comment:
Looks like there's a few more mouseOver/mouseOuts in this file to be converted (that aren't actually breaking tests, but we should change)